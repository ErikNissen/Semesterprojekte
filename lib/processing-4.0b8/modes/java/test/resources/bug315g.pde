version https://git-lfs.github.com/spec/v1
oid sha256:d123a7fe4eb291d430abbb7fef6a84114487a57fd09a464040e6499c1ab062ec
size 78
