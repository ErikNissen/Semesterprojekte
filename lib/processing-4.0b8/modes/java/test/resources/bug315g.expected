version https://git-lfs.github.com/spec/v1
oid sha256:d089736ec3fa3c9a191f0e79b4d5777e1e86a1d5d826a8f68e7a4ea89b80316e
size 887
